homeassistant/components/history.py

```diff
@@ -283,10 +291,11 @@ def get(self, request, datetime=None):
         else:
             end_time = start_time + one_day
         entity_id = request.query.get('filter_entity_id')
+        get_initial_state = 'skip_initial_state' not in request.query
```

Comment:
I don't see this functionality being used in the frontend PR.

Reply:
https://github.com/home-assistant/home-assistant-polymer/pull/432/files#diff-9a760fd0055ca6db1d1353c9e7f3d16cR321